src/lib/exporter/ffmpegExporter.ts

```
@@ -0,0 +1,408 @@
+				framerate: this.config.frameRate,
+				hardwareAcceleration: "prefer-hardware",
+				avc: { format: "annexb" }, // CRITICAL: FFmpeg raw h264 pipe needs Annex B format with start codes!
+			});
+
+			// 7. Decode & render frames, pipe directly from GPU encoder
+			let frameIndex = 0;
+			const exportStartTime = Date.now();
+			const frameDurationUs = 1_000_000 / this.config.frameRate;
+
+			// Add webcam queue
+			const webcamFrameQueue = this.config.webcamVideoUrl ? new AsyncVideoFrameQueue() : null;
```

Biome (2.4.10)

[error] The AsyncVideoFrameQueue variable is undeclared.

(lint/correctness/noUndeclaredVariables)